DEVELOPER_GUIDE_OLD.md

```diff
@@ -400,6 +400,7 @@ Add the required configurations to the  <APIM_HOME>/repository/conf/api-manager.
 
     <AISecurityHandler>
         <ApplyForAllAPIs>false</ApplyForAllAPIs>
+        <ValidateCerts>True</ValidateCerts>
```

Comment:
Validate certs should be default behavior. so this property should be something like SkipCertValidate, InsecureClientMode, etc

Reply:
Default behavior would be having true to this config. Only if the user needs to skip validation, has to add false. Only the config has to be included in the false scenario.

Reply:
Changed the config to SkipCertValidation